src/linear_algebra/linear_independent.lean

```diff
@@ -176,6 +176,10 @@ begin
   rw [finsupp.map_domain_apply hf]
 end
 
+lemma linear_independent.reindex_range (i : linear_independent R v) :
```

Comment:
```suggestion
lemma linear_independent.coe_range (i : linear_independent R v) :
```
If you agree with this suggestion, please update the PR title.

Reply:
I want the name to match `basis.reindex_range`. I'll add a comment to this effect.

Reply:
On second thoughts, your suggestion is better.